api/envoy/api/v2/core/socket_option.proto

```diff
@@ -0,0 +1,47 @@
+syntax = "proto3";
+
+package envoy.api.v2.core;
+
+import "validate/validate.proto";
```

Comment:
You'll need to add option `(udpa.annotations.file_migrate).move_to_package = "envoy.config.core.v3";` (as done in other files in this package) for the correct file movement in v3.
`